webapp/public/run.html

```diff
@@ -184,6 +185,25 @@
                     // propagate message packets to parent frames
                     window.parent.postMessage(msg.data, "*");
                 }
+                else if (server && data.type === "kiosk") {
```

Comment:
```suggestion
                else if (server && data.type === "builtjs") {
```

 might be a bit more descriptive / findable?